index.html

```diff
@@ -173,5 +172,17 @@ <h5>Skills</h5>
             </li>
         </ul>
     </section>
+    <div class="form-container">
+        <form action="https://formspree.io/f/xyyovkvo" method="post">
+            <h2>Contact me</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            <input type="text" id="Name" name="Name" placeholder="Your name.." minlength="3" maxlength="30" required>
+            <input type="email" id="email" name="email" placeholder="  Your e-mail.." required>
+            <textarea id="message" name="subject" maxlength="500" placeholder="Write your message here" required></textarea>
+            <input type="submit" value="Get in touch">
+        </form>
+        <div class="endline">
+        </div>
+    </div>
```

Comment:
- [ ] Great work adding a background image to the form section but the Header _**Contact Me**_ is being overlapped by the arc of the background image. Therefore, the header is not readable. Kindly adjust the position of the background image to be similar to that of the Figma design that you chose. 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/172705957-14ca77bb-c4cc-4c3d-90f0-7050db0c1480.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/172706195-64bf6ed4-8c8b-42e1-93b0-d008b3aadb6a.png)

You can read this [article](https://www.w3schools.com/cssref/pr_background-position.asp) to have an idea of how to adjust the position. 👍

Reply:
- Optional
@TemesgenA , i would recommend adding a margin between the about section and contact section. Also there is some margin/padding on the left and right sides throughout your application. I would recommend you remove it, i am sure you would like it way better that way.
Here is a snapshot of how your application looks now.
![image](https://user-images.githubusercontent.com/33268927/172993179-bb728053-5dbb-4946-ae95-d665bdd0be80.png)